README.md

```diff
@@ -0,0 +1,98 @@
+* Using axios to read data from an API in JavaScript
+* Parsing through an API response in JavaScript and extracting desired information
+* Using axios to write data to an API in JavaScript
+
+## Introduction
+
+Ada Lovelace is starting a pet rescue service advertising pets in need of rescue via an API and is building a CLI app.   This app should load the list of pets automatically and present a menu with options.   The user should be able to list all pets, select a pet and see details, add new pets and remove a pet from the App once adopted.
+
+She's managed to figure out the CLI parts on her own but needs your help with the API requests.
+
+You will be using the [Pets API](https://github.com/AdaGold/pets-api) for this.
+
+### Setup
+
+1. Fork and clone this repository
+2. Run `npm install` to install dependencies
+3. Run the app with `node src/main.js`
+
+### Layout
+
+You will be adding your code to `src/adaPets.js`.  All of the functions you need to implement are currently there, just with empty bodies.
+
+There are two other files that we've provided that you won't need to change `src/main.js` and `src/result.js`.  They are worth taking a quick look at but don't worry too much if you don't understand them fully.
+* `src/main.js` is the actual driver code for the CLI app.  If you want to manually test your code you can use this.
+* `src/result.js` is the library we've written to handle results and errors for this app.  It provides three functions, two of which you will use and one of which you're going to see in the tests.  **Every helper function should make sure to call `setResult` or `setError` before each `return`.**
+  * `setResult` call this function to set the result of your function.  We use this because Axios is asynchronous.
+  * `setError` call this function to indicate an error in your function.  Again, we use this because Axios is asynchronous.
+  *  If it takes too long for your code to produce a result (like say because you haven't written it yet) it will time out.
+
+There are tests for each of the waves in the `test/` directory, there is one file per wave.  For example you can run the Wave 1 tests with `jest test/wave1.test.js`.
+
+**Note: Because Axios is asynchronous we will need to use `setResult` instead of returning results and `setError` instead of throwing errors.**
+
+## Wave 0: Familiarize Yourself with the Pets API
+
+Read through the [documentation for the Pets API](https://github.com/AdaGold/pets-api) and experiment with using it via Postman so you can have a handle on how it works for the tasks below.
+
+## Wave 1: List Pets
+
+Before we start doing things like looking at individual pets we want to be able to get a list of all of the pets.
+
+To do this fill out the `listPets` function.  This will need to make a call to the Pets API and should `setResult` a list of objects containing (at least) the pet's `id` and `name` and should call `setError` with an error message if the request fails.
```

Comment:
My comments are all out of the place :)

Anyway, everything else still holds: every wave maybe there can be a bulleted list about the expectations of `listPets` (so students can think of it as a checklist)

Also, Becca just tried to see what the error object came back for `error.response.data` from the API as `undefined`... Not sure what the state of the Pets API is